mlflow/utils/autologging_utils/safety.py

```diff
@@ -621,22 +664,19 @@ def update_wrapper_extended(wrapper, wrapped):
     return updated_wrapper
 
 
-def _wrap_patch(destination, name, patch, settings=None):
+def _wrap_patch(destination, name, patch_obj, settings=None):
     """
-    Apply a patch while preserving the attributes (e.g. __doc__) of an original function.
+    Apply a patch.
 
     :param destination: Patch destination
     :param name: Name of the attribute at the destination
-    :param patch: Patch function
+    :param patch_obj: Patch object
```

Comment:
Can we clarify what types are expected & indicate that this isn't an instance of `gorilla.Patch`?